modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImplTest.java

```diff
@@ -241,7 +241,8 @@ public void init(TestInfo info) {
     }
 
     private void setupCluster(CacheFactory mappingCacheFactory, Function<String, QueryTaskExecutor> executorsFactory) {
-        DdlSqlToCommandConverter converter = new DdlSqlToCommandConverter();
+        // Pass null as logical topology service because zone creation isn't a test target there and will be never called.
```

Comment:
this comment doesn't seems to be relevant

Reply:
Fair! Removed the commentary.